interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformer.scala

```diff
@@ -0,0 +1,73 @@
+package io.buoyant.transformer.k8s
+
+import com.twitter.finagle.Name.Bound
+import com.twitter.finagle.{Addr, Name, Address, NameTree}
+import com.twitter.util.{Var, Activity}
+import io.buoyant.namer.{DelegateTree, DelegatingNameTreeTransformer}
+
+/**
+ * The DaemonSetTransformer maps each address in the NameTree to a member of
+ * a given daemon set that is on the same /24 subnet.  Since each k8s node
+ * is its own /24 subnet, the result is that each address is mapped to the
+ * member of the daemon set that is running on the same node.  This can be used
+ * to redirect traffic to a reverse-proxy that runs as a daemon set.
+ */
```

Comment:
this comment is a little unclear -- there are multiple name trees.  What is daemonSet and how is it used?